Cargo.toml

```diff
@@ -117,6 +117,6 @@ tao-macros = { version = "0.1.0" }
 [dev-dependencies]
 http-range = "0.1.5"
 pollster = "0.3.0"
-tao = { version = "0.24", default-features = false, features = [ "rwh_05" ] }
-wgpu = "0.18.0"
-winit = { version = "0.29", features = [ "rwh_05" ] }
+tao = { version = "0.24", default-features = false, features = [ "rwh_06" ] }
```

Comment:
I was not sure this line should be simplified to `tao = "0.24"` so I preserved the configuration as much as possible.

Reply:
let's simpify it

Reply:
Sure.